packages/manager/modules/core/src/translate/translate.constants.js

```diff
@@ -0,0 +1,5 @@
+export const HEADER_LOCALE = 'Content-Language';
```

Comment:
Shouldn't we have a custom header instead ? It would give us more control !

Reply:
(I'm kidding)

Reply:
😭